src/ArrayToXml.php

```diff
@@ -13,6 +13,8 @@ class ArrayToXml
 
     protected $replaceSpacesByUnderScoresInKeyNames = true;
 
+    protected $dropXmlDeclaration = false;
```

Comment:
Let's make this positive by renaming this to `addXmlDeclaration` and set it to `true` by default